fitsmap/convert.py

```diff
@@ -912,14 +915,20 @@ def tile_markers(
         pbar_ref, unit="zoom levels", total=max_zoom + 1 - min_zoom
     )
 
+    if cluster_radius is None:
+        cluster_radius = max(max(max_x, max_y) / tile_size, 40)
+    if node_size is None:
```

Comment:
`node_size` here should be `cluster_node_size` I believe.

Reply:
ohh! Thanks for the check. I will fix it.